rnd/infra/helm/autogpt-server/values.dev.yaml

```diff
@@ -82,6 +82,6 @@ env:
   APP_ENV: "dev"
   PYRO_HOST: "0.0.0.0"
   NUM_GRAPH_WORKERS: 100
-  NUM_NODE_WORKERS: 100
+  NUM_NODE_WORKERS: 5
```

Comment:
This is to prevent the server from blowing up, as `multiprocessing.Pool` doesn't dynamically spawn processes based on need but instead spawns all of them on init. @aarushik93